src/Integration/Binding/SolutionBindingOperation.cs

```diff
@@ -177,15 +177,15 @@ public void Initialize(IEnumerable<Project> projects, IDictionary<Language, Sona
 
             foreach (var project in projects)
             {
-                if (projectBinderFactory.Get(project) != null)
+                if (projectBinderFactory.Get(project) is CFamilyProjectBinder)
```

Comment:
This should eventually move to ProjectBinder.

Reply:
How are you tracking the bits of code that should change? You've called out a number of them in review comments, but they will be effectively lost once the PR is merged.

If I'm working on a feature/refactoring that I know is going to be split across multiple sprints and have a something I want to revisit in a later commit, I normally put a comment in the code with `duncanp` in it.
I find it works better for me than a `TODO`, and it's easy to check that removed them all before committing.

Reply:
I write in a notebook :) But I also periodically check the base branch and evaluate how everything looks.

Reply:
Ok, but writing in a notebook and having the information in your head doesn't help anyone else in the team if they have to pick up the work for any reason. It's better to have some kind of public/shared record of what still needs to be done.